ui/app/models/pki/role.js

```diff
@@ -219,6 +227,20 @@ export default class PkiRoleModel extends Model {
   })
   ou;
 
+  @attr('array', {
+    defaultValue() {
+      return ['DigitalSignature', 'KeyAgreement', 'KeyEncipherment'];
+    },
+  })
+  keyUsage;
+
+  @attr('array', {
+    defaultValue() {
+      return [];
+    },
+  })
+  extKeyUsage;
```

Comment:
Nice! Did you add this in case this attr isn't a consistent type as well?

Reply:
Yeah, I needed to define this attribute as an array because without it, trying to set the value to an array was falling back to just the string. Probably some updates we can make in `openApiToAttrs` to address this but figured that was out of scope for this